lib/av_client/crypto/util.ts

```diff
@@ -0,0 +1,75 @@
+import * as crypto from "../aion_crypto";
+import * as sjcl from "../sjcl";
+
+// As this is working with untyped SJCL classes,
+// we need the _any_ type in this wrapper.
+/*eslint-disable @typescript-eslint/no-explicit-any*/
+
+
+export const Curve = crypto.Curve;
+
+// Converter functions
+// --------------------------
+export const pointFromBits = (bits) => crypto.pointFromBits(bits);
+
+export const pointFromHex = (hex: string): Point => new Point(pointFromBits(sjcl.codec.hex.toBits(hex)));
+export const pointToHex = (point: Point): string => sjcl.codec.hex.fromBits(point.toBits(true));
+
+export const bignumFromHex = (hex: string): Bignum => new Bignum(sjcl.bn.fromBits(sjcl.codec.hex.toBits(hex)));
+export const bignumToHex = (bignum: Bignum): string => sjcl.codec.hex.fromBits(bignum.toBits());
+
+export const hashToBignum = (hash: BitArray): Bignum => new Bignum(crypto.hashToBn(hash));
+
+// Other
+// --------------------------
+export const generateRandomBignum = () => new Bignum(crypto.randomBN());
+
+export const pointFromX = (x: Bignum): Point => {
```

Comment:
This should be covered by tests.

Reply:
It is tested by `commitment.test.ts`. There's a ~99% coverage of the code in the PR.

Reply:
I must have overlooked it. Thanks.